src/util/ui.js

```diff
@@ -22,6 +22,19 @@ export function ellipsis(value) {
   return value;
 }
 
+export function ellipsisCollectorAddress(value) {
+  if (value) {
+    const len = value.length;
+    const dots = '...';
+    if (!value) return '';
+    if (value.length > 10) {
+      return value.substring(0, 8) + dots;
+    }
+    return value;
+  }
+  return value;
+}
+
```

Comment:
Can reuse this instead?
https://github.com/likecoin/liker-land/blob/447460d12ebaa6cfbd5acfdc90426048875a72ea/src/util/ui.js#L12-L23

Reply:
[♻️ Reuse ellipsis functions for memo name](https://github.com/likecoin/liker-land/pull/2002/commits/55d74767618c7223f64c160fa73e351cd2bcf721)